bundle/deploy/metadata.go

```diff
@@ -0,0 +1,18 @@
+package deploy
```

Comment:
Please move to a new "deployment" package. The other things in this package pertain to the act of deploying.

Reply:
done, makes sense!